tests/functional/aws-node-sdk/test/object/objectCopy.js

```diff
@@ -149,6 +170,64 @@ describe('Object Copy', () => {
                 );
         });
 
+        it('should copy an object from a source bucket to a different ' +
+            'destination bucket and copy the tag set if no tagging directive' +
+            'header provided', done => {
+            s3.copyObject({ Bucket: destBucketName, Key: destObjName,
+                CopySource: `${sourceBucketName}/${sourceObjName}` },
+                err => {
+                    checkNoError(err);
+                    checkSuccessTagging(originalTagKey, originalTagValue, done);
+                });
+        });
+
+        it('should copy an object from a source bucket to a different ' +
+            'destination bucket and copy the tag set if COPY tagging ' +
+            'directive header provided', done => {
+            s3.copyObject({ Bucket: destBucketName, Key: destObjName,
+                CopySource: `${sourceBucketName}/${sourceObjName}`,
+                TaggingDirective: 'COPY' },
```

Comment:
can we do a test where you say 'COPY' but send new tags anyway to make sure the tags are ignored or the correct error is returned?